src/config/baselines.tsx

```diff
@@ -0,0 +1,55 @@
+import nsoDisabled from '../data/nso/NSO_Disabled_Admin1_Total.json';
+import nsoHerders from '../data/nso/NSO_Herder_HHs_Admin2.json';
+import nsoHerdsize from '../data/nso/NSO_Herd_Size_Admin1_LT_200.json';
+import nsoChild from '../data/nso/NSO_Child_U5_Admin2.json';
+import nsoLivestock from '../data/nso/NSO_Livestock_Count_ths_Admin2.json';
+import nsoElderly from '../data/nso/NSO_Single_Elderly_Admin1_Total.json';
+import nsoPoverty from '../data/nso/NSO_Poverty_Headcount_Admin1.json';
+import nsoPop from '../data/nso/NSO_Population_Admin2_Total.json';
+
+export interface NsoData {
+  TBL_ID: string;
+  Period: string;
+  CODE?: string;
+  SCR_MN: string;
+  SCR_ENG: string;
+  CODE1?: string;
+  CODE2?: null;
```

Comment:
Is this really `null` typed? This type definition literally says `CODE2` will be either `undefined` or `null`, was that the intention?

Reply:
string, typo